### DISCUSSION TITLE
Icon request: icon-hotairballoon

### DISCUSSION THREAD

**Discussion by ElbieLouw** [FortAwesome/Font-Awesome#21127] [Icon Requests]
Hi there

I have a client that does hot air balloon flights and I would LOVE to put a little icon next to the menu option

Regards
Elbie

**Comment by shaedrich** [comment]
Maybe something like this?
http://www.flaticon.com/free-icon/hot-air-balloon_66410

**Comment by ktuley** [comment]
Bumping this. I'd Love not only a Hot Air balloon which is more light bulb shaped AND a Gas balloon which is more round, the one suggested by Shaedrich

**Comment by shaedrich** [comment]
It's baffling that I changed companies three times since I requested this icon for a software, the company I was working at that time, develops, and FontAwesome hasn't bothered to consider adding this not at all unusual icon

**EDIT:** But it's nice to know that you're now refining your icon request workflow :)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by ktuley** [comment]
Yippee!!! great news. Thanks for letting me know


—————
Ken Tuley
***@***.***
